foundry.toml

```diff
@@ -70,4 +70,8 @@ out = 'packages/contracts-da-bridge/foundry_artifacts'
 ffi = true
 fuzz-runs = 10_000
 
-
+[profile.ops]
+src = 'packages/contracts-ops/contracts'
+test = 'packages/contracts-ops/contracts/test'
+out = 'packages/contracts-ops/foundry_artifacts'
+ffi = true
```

Comment:
- Do we need ffi? 
- I think we need file permissions

Reply:
gonna leave ffi in in case I want JQ